python/dgl/graphbolt/impl/fused_csc_sampling_graph.py

```diff
@@ -505,8 +505,9 @@ def sample_neighbors(
         fanouts: torch.Tensor,
         replace: bool = False,
         probs_name: Optional[str] = None,
+        # TODO: clean up once the migration is done.
```

Comment:
Move this above L510, I don't think we need to clean up `deduplicate`, right?

Reply:
> Move this above L510, I don't think we need to clean up `deduplicate`, right?

After migration, both unique and not_deduplicate use csc_foramt sampled subgraph, so we don't need `deduplicate` to select which function to use.